docs/index.md

```diff
@@ -158,8 +158,41 @@ When an issuer generates a new key to sign Health Cards, the public key SHALL be
 issuer's JWK set in its `jwks.json` file. Retired private keys that are no longer used to sign Health Cards SHALL be destroyed.
 Older public key entries that are needed to validate previously
 signed Health Cards SHALL remain in the JWK set for as long as the corresponding Health Cards
-are clinically relevant. However, if a private signing key is compromised, then the issuer SHALL immediately remove the corresponding public key
-from the JWK set in its `jwks.json` file and request revocation of all X.509 certificates bound to that public key.
+are clinically relevant. However, if a private signing key is compromised, then the issuer SHALL immediately remove the corresponding public key from the JWK set in its `jwks.json` file and request revocation of all X.509 certificates bound to that public key; verifiers will from then on reject all Health Cards signed using that key.
+
+### Revocation
+
+Individual Health Cards MAY be revoked by an issuer using the digest of their JWS representation. The Revocation Digest Fragment (RDF) of a Health Card is calculated as follows:
+1. Calculate the SHA-256 `<<digest>>` of the utf-8 encoding of the `<<Health Card as JWS>>`,
+2. Calculate the base64url encoding of the first 128 bits of resulting `<<digest>>`
+
+To enable per-card revocation, the issuer creates, for each of its keys, a JSON Card Revocation List (CRL) file with the following content:
+```json
+{
+"kid": "<<kid>>",
+"ctr": "<<ctr>>",
+"rdf": [...]
+}
+```
+where
+- `"<<kid>>"` is the ID of the corresponding issuer key,
+- `"<<ctr>>"` is counter indicating how many times this file has been updated; initial value is 1,
+-  `rdf` is an array of revoked cards' digest fragments.
```

Comment:
I have a minor complaint regarding the use of RDF because it could be confused with RDF/JSON serialization (https://www.w3.org/TR/rdf-json/).  Furthermore, it is overloaded to refer to a single revocation digest fragment and used as the field name for the array of RDFs.  The latter could be fixed by renaming the field "rdfs", but due to my first point I think a better term could be used.  Perhaps SHC Thumbprint?  Then the CRL field might be "shcts".

Reply:
Wasn't aware of the w3 RDF, we certainly want to avoid collisions.